File: copl/com.elster.jupiter.unifyingjs/src/main/web/js/uni/src/override/form/field/Text.js
/*
 * Copyright (c) 2017 by Honeywell International Inc. All Rights Reserved
 */

Ext.define('Uni.override.form.field.Text', {
    override: "Ext.form.field.Text",
    blankText: Uni.I18n.translate('fieldValidation.blankText', 'UNI', 'This field is required')
});

File: copl/com.elster.jupiter.util/src/main/java/com/elster/jupiter/util/conditions/Text.java
/*
 * Copyright (c) 2017 by Honeywell International Inc. All Rights Reserved
 */

package com.elster.jupiter.util.conditions;

public class Text extends Leaf {
	
	private final String text; 
	
	Text(String text) {
		this.text = text;
	}
	
	public String getText() {
		return text;
	}
	
	@Override
	public void visit(Visitor visitor) {
		visitor.visitText(this);		
	}
	
	@Override 
	public String toString() {
		return "Text: " + text;
	}
}
